sdk/template/ci.yml

```diff
@@ -9,12 +9,7 @@ trigger:
   paths:
     include:
     - sdk/template/
-#if (false)
```

Comment:
Don't remove the if block. It is useful comment section.

Reply:
In fact we need to add back the if block that was removed as part of your other PR. @chunyu3 is there some reason you are trying to remove these?

Reply:
See my comment in your other PR https://github.com/Azure/azure-sdk-for-net/pull/28016#discussion_r852480070.

Reply:
I updated according to the comment and add eng/common in pr-trigger the same as trigger does